py/fetcher/main.py

```diff
@@ -0,0 +1,160 @@
+import certifi
```

Comment:
What do you need this for? That would be another dependency you need to make everybody import.

Reply:
It was a workaround for a macOS issue, moved the logic [here](https://github.com/getsentry/usage-accountant/blob/feat/ddog-metrics-support/py/fetcher/datadog.py#L59).
I tested in a sk pod, the workaround is NOT needed in there, so will remove it before merging to `main`.